src/serve_rendered.js

```diff
@@ -1020,8 +1020,23 @@ export const serve_rendered = {
                 const format = extensionToFormat[extension] || '';
                 createEmptyResponse(format, '', callback);
               }
+            } else if (protocol === 'file') {
+              const file = url.fileURLToPath(req.url);
```

Comment:
This didn't work for me testing locally on windows

Locally I changed to
```
              const dir = options.paths[protocol];
              const name = unescape(req.url).substring(protocol.length + 3);
              const file = path.join(dir, name)
```
which did seem to work

Reply:
I will apply that change later, sorry, I have no Windows available for testing.

Reply:
This is what the error looks like with the default code. *note* I added a console.log after `const file = url.fileURLToPath(req.url);` just so I could see the path, so you can see the file path is shown in the console multiple times `\\file_65596_2024-07-22_013451.geojson\`
![image](https://github.com/user-attachments/assets/e643ed2a-64d5-4b8a-ac44-0b8a865edc1a)

Reply:
my suggested code ends up with a path like this. The top path is 'dir' and the second path is 'file'
![image](https://github.com/user-attachments/assets/397bb00b-3555-47c9-8db3-df65bd3258d8)

Reply:
I applied your change and dropped my idea of using "file://" as real uri, because it would allow to expose the whole filesystem.